src/rosdiscover/observer/observer.py

```diff
@@ -35,17 +51,17 @@ def for_container(cls,
             yield ROS2Observer(instance, config)
 
     @abstractmethod
```

Comment:
@abstractmethod isn't necessary here. (Since this is the constructor that's actually used by ROS1Observer and ROS2Observer, it's quite likely that the program will fail upon attempting to construct an instance of one of those two classes.)

Reply:
Done